src/assets/i18n/custom-translations.json

```diff
@@ -178,6 +178,7 @@
   "lineListingSharedDisease": "Shared disease",
   "lineListingSharedLocation": "Shared location",
   "headingUserProfile": "Role name",
-  "confirmSetMissingGeoCoordinates": "Also replace existing coordinates",
-  "warningSetMissingGeoCoordinates": "This might replace coordinates which were intentionally set differently!"
+  "confirmSetMissingGeoCoordinates": "Also replace exising coordinates",
+  "warningSetMissingGeoCoordinates": "This might replace coordinates which were intentionally set differently",
+  "caseSpecificSearch": "Seach by Case ID, External ID, epid number"
```

Comment:
i think it's better to use the already existing text for this (strins.promptSearch) instead of adding a new one

Reply:
Resolved.